.gitignore

```diff
@@ -1,6 +1,7 @@
 /.vscode/*
 !/.vscode/launch.json
 !/.vscode/tasks.json
+.idea/
```

Comment:
What is the .idea/?

Reply:
Oh, I was using Webstorm, .idea/ is similar to .vscode/ in Visual Code. Will remove this.